README.md

```diff
@@ -13,6 +12,8 @@ If you feel you can not, or do not want to ask questions on [EngineerZone](https
 
 There is no free replacement for consulting services. If you have questions that are best handed one-on-one engagement, and are time sensitive, consider hiring a consultant. If you want to find a consultant who is familar with the HDL found in this repository - ask on [EngineerZone](https://ez.analog.com/community/fpga).
 
+Analog Devices will provide limited online support for anyone using the core with Analog Devices components (ADC, DAC, Clock, etc) via [EngineerZone](https://ez.analog.com/community/fpga) under the GPL license. If you would like deterministic support when using this core with an ADI component, please investigate a commercial license. Using a non-ADI JESD204 device with this core is possible under the GPL, but Analog Devices will not help with issues you may encounter.
+
```

Comment:
I don't think we should add this line

Reply:
Removed this commit (_README.md: Update "support" text_)